.github/CODEOWNERS

```diff
@@ -281,12 +271,11 @@
 /sdk/python/jobs/single-step/tensorflow/train-hyperparameter-tune-deploy-with-tensorflow/train-hyperparameter-tune-deploy-with-tensorflow.ipynb @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
 /sdk/python/resources/compute/compute.ipynb @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
 /sdk/python/resources/workspace/workspace.ipynb @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
-/sdk/python/resources/workspace/workspace.ipynb @joshharrin @seanyao1 @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  

```

Comment:
again, leave this line as it was.  We don't want to delete when names are added to ours.